src/pkg/path/path_test.go

```diff
@@ -749,3 +749,61 @@ func (suite *PathUnitSuite) TestPath_piiHandling() {
 		})
 	}
 }
+
```

Comment:
can you also make a test for calling `Folders()` on a `ServicePrefix` path? I'm not sure if that will throw an index out of bounds error right now since it's kind of escaping the original assumptions of Path

Reply:
good idea